wazo_confd/plugins/api/api.yml

```diff
@@ -116,6 +116,11 @@ parameters:
     type: string
     description: Search term for filtering a list of items. Only items with a field
       containing the search term will be returned.
+  query_string_uuid_filter:
+    name: uuid
+    in: query
+    type: string
+    description: Search filter for resource UUID, can be used to search multiple elements. Using a comma separated list of UUID
```

Comment:
For the API part, I'd propose adding a few parameters to make the swagger interface more intuitive
```
    required: false
    collectionFormat: csv
    minItems: 1
    maxItems: 20
    uniqueItems: true
    type: array
    items:
      type: string
      format: uuid
```